presidentielles/sondages.json

```diff
@@ -0,0 +1,2094 @@
+                        "Nicolas Dupont-Aignan": 4,
+                        "Florian Philippot": 1,
+                        "Marine Le Pen": 24,
+                        "François Asselineau": 1,
+                        "Jean Lassalle": 1
+                    }
+            },
+            "ciotti": {
+                    "intentions_exprimees": null,
+                    "intentions": {
+                        "Nathalie Arthaud": 1,
+                        "Philippe Poutou": 1,
+                        "Fabien Roussel": 3,
+                        "Jean-Luc Mélenchon": 12,
+                        "Anne Hidalgo": 7,
+                        "Yannick Jadot": 6,
+                        "Emmanuel Macron": 30,
+                        "Jean-Christophe Lagarde": 0,
+                        "Éric Ciotti": 6,
+                        "Nicolas Dupont-Aignan": 4,
+                        "Florian Philippot": 2,
+                        "Marine Le Pen": 26,
+                        "François Asselineau": 1,
+                        "Jean Lassalle": 1
+                    }
+            },
+            "montebourg": {
+                    "intentions_exprimees": null,
+                    "intentions": {
+                        "Nathalie Arthaud": 1,
+                        "Philippe Poutou": 1,
+                        "Fabien Roussel": 2,
+                        "Jean-Luc Mélenchon": 13,
+                        "Arnaud Montebourg": 5,
+                        "Yannick Jadot": 6,
+                        "Emmanuel Macron": 25,
+                        "Jean-Christophe Lagarde": 0,
+                        "Xavier Bertrand": 16,
+                        "Nicolas Dupont-Aignan": 4,
+                        "Florian Philippot": 1,
+                        "Marine Le Pen": 24,
+                        "François Asselineau": 1,
+                        "Jean Lassalle": 1
+                    }
+            }
+        },
+        "second_tour": {
+            "lepen_macron": {"intentions_exprimees": null, "intentions": {"Emmanuel Macron": 55, "Marine Le Pen": 45}}
+        },
+        "reports": {
+            "bertrand": {
+                "candidats_1er_tour": ["Jean-Luc Mélenchon", "Anne Hidalgo", "Yannick Jadot", "Emmanuel Macron", "Xavier Bertrand", "Marine Le Pen"],
+                "candidats_2nd_tour": ["Marine Le Pen", "Emmanuel Macron", "Abstention/Blanc/Nul"],
+                "reports": [
+                    [34, 19, 47],
+                    [33, 9, 58],
+                    [43, 13, 44],
+                    [97, 1, 2],
+                    [37, 29, 34],
+                    [2, 97, 1]
+                ]
+            }
+        }
+     },
+
+    "20210822_ipsos": {
+        "institut": "Ipsos",
+        "commanditaires": ["Libres!"],
+        "date_debut": "2021-08-20",
+        "date_fin": "2021-08-22",
+        "date_publication": "2021-08-27",
+        "echantillon": 938,
+        "note": "Certains d'aller voter",
```

Comment:
Shouldn't it be `sample_type` instead of `note`?

Reply:
Yeah nothing is consistent past the first poll. I'm making sure the last one has the right structure, and then I'll correct all previous polls until CI passes.